frontend/src/actions/KingdomActions.js

```diff
@@ -0,0 +1,40 @@
+import * as actions from '../constants/ActionTypes';
+import { fetchByKingdom } from '../services/fetchService';
+
+export const setKingdomAction = kingdomID => {
+  return dispatch => {
+    dispatch({
+      type: actions.TYPE,
```

Comment:
@kkeszthe why? you can store this in a constant, but it should be different for each action (even if you dont do anything with it in a reducer) so you can see in redux what's happening